composer.json

```diff
@@ -25,7 +25,7 @@
 			"@php -r \"file_exists('local-config.php') || copy('local-config-sample.php', 'local-config.php');\"",
 			"@php -r \"file_exists('local-config.json') || copy('local-config-sample.json', 'local-config.json');\""
 		],
-		"setup-wordpress": "./vendor/bin/wp core download --version=6.2.2 --quiet --skip-content --force",
+		"setup-wordpress": "./vendor/bin/wp core download --version=6.3-RC3 --quiet --skip-content --force",
```

Comment:
Good call testing the 6.3 RCs. Let's hold on this until 6.3 comes out (today?!). and update to that as part of a chore branch.

Reply:
I've reverted this to 6.2.2 in this branch. We'll update to 6.3 separately.